evap/staff/templates/staff_user_form.html

```diff
@@ -40,11 +40,22 @@ <h5 class="card-title">{% trans 'CC-User for' %}</h5>
             </div>
             <div class="card mb-3">
                 <div class="card-body">
-                    <h5 class="card-title">{% trans 'Courses contributing to (active semester)' %}</h5>
+                    <h5 class="card-title">{% trans 'Personal result pages' %}</h5>
                     {% if courses_contributing_to %}
```

Comment:
the if should be around the card so that it is only shown if there are courses the user contributed to